infrastructure/cdn-in-a-box/README.md

```diff
@@ -0,0 +1,41 @@
+# CDN-In-A-bOx (CIAO) - Notes
+
+## Build step
+So far, the build is done from this directory (/infrastructure/cdn-in-a-box/), using the following command:
+
+```bash
+docker-compose -f ./docker-compose.yml -f traffic_ops/docker-compose.yml -f traffic_ops_db/docker-compose.yml -f traffic_portal/docker-compose.yml up --build
+```
+
+This will build and start all of the currently-implemented pieces of CIAO (note that you will likely need to be root or use `sudo` to communicate with the `docker` daemon). By default, it leaves your terminal's stdout open to the logs produced by various parts of the CDN.
+
+
+## <a name="ports"></a> Ports and Interfaces
+CIAO provides API gateways and user interfaces over HTTP(S) for the pieces of the CDN that support it. By default, all of these services bind to the local address `0.0.0.0`, and are spread out over various ports. Here's a list of the ports and the services they provide:
+
+* 443 - Exposes the Traffic Ops landing page, from which you may log in and view/manage/manipulate your CIAO at will (See [the Traffic Ops documentation](http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html) for details). This is also the endpoint for the Go-based Traffic Ops API, and will handle all REST API requests (including those intended to be processed by Perl).
+
+* 5432 - Exposes a connection to the Traffic Ops's Postgres database. Note that this is *not* meant to be used through a browser, and instead is an endpoint for direct connection to the database (this is used by Traffic Ops to store information).
+
+* 8080 - The "adminer" portal which allows access to the Traffic Ops Database through a browser.
+
+* 60443 - This is the endpoint for the old Perl-based API for Traffic Ops. Rather than send requests here, you should send them to port 443, as the Go-based API there will act as a reverse proxy to pass off API requests it doesn't directly handle back to this port.
+
+* 61443 - The Traffic Portal landing page. Generally-speaking, this is much easier to use and far better-looking than the Traffic Ops interface, so use of the former is recommended over that of the latter.
```

Comment:
Move this to the first bullet point and explicitly state "This is the port you should use to manage the CIAO"

Reply:
thinking I should perhaps just make TP port 443.